nearpy/tests/engine_tests.py

```diff
@@ -73,5 +74,53 @@ def test_retrieval_sparse(self):
             self.assertEqual(y_data, x_data)
             self.assertAlmostEqual(y_distance, 0.0, delta=delta)
 
+    def get_keys(self, engine):
+        for lshash in engine.lshashes:
+            bucket = engine.storage.buckets[lshash.hash_name][lshash.hash_name]
+            return {i[1] for i in bucket}
```

Comment:
Aw, you're right... It was fine before. I will take a look later on, ok? Thanks for your help!